### ISSUE TITLE
Can't make "Biserial" association analysis work

### ISSUE THREAD

**Issue by maialenzaze** [microbiome/microbiome#211]
Hello there,

I am trying to replicate the "Cross-correlating data sets" in "Association Analysis". The issue with my data is that instead of a continuous lipids matrix, I have a dichotomous adjacency matrix filled with 1 and 0, and therefore, the "spearman" correlation is not the most appropriate. I have tried changing the method to "Pearson", for which it doesn't find any significant correlation across my data. I would like to try running a point-biserial correlation to explore any possible correlations in my data. However, when I replace the method with "biserial" it seems that it exists in the association function, but an argument is missing.

> correlations <- associate(x, y, method = "biserial", mode = "matrix", p.adj.threshold = 0.05, n.signif = 1)
Error in associate(x, y, method = "biserial", mode = "matrix", p.adj.threshold = 0.05,  : 
  object 'inds' not found

Do you know any way to make the biserial correlation analysis work?

Thank you very much in advance.

**Comment by antagomir** [comment]
Isn't the biserial correlation for binary data equivalent to Pearson correlation on binary data?

**Comment by maialenzaze** [comment]
Thank you very much for your response.

I had understood that Pearson is used when both variables are continuous, and that Point-Biserial is a special case of Pearson where one of the variables is binary. I did not know about the methodological difference between both. When I saw that "biserial" could exist in the function I tried to run that instead of "pearson". For what I've read on the Internet, Point-biserial seems to follow the same methodology as Pearson, so problem solved.
